src/main/java/com/univocity/parsers/containers/Column.java

```diff
@@ -0,0 +1,10 @@
+package com.univocity.parsers.containers;
+
+/**
+ * @author naveen.kasthuri
+ * A column reference. Only enums should implement this interface.
+ * Column header must match enum such that columnHeaderEnum.toString().equals(columnHeader) is true.
+ * @see com.univocity.parsers.examples's RowObjectExample for an implementation example
+ */
+public interface Column {
```

Comment:
If you require implementations of Column to be enumerations, you must enforce this in the interface definition with generics:

```
    public interface Column <T extends Enum<T>> {
```